modulefiles/GDAS/wcoss2.intel.lua

```diff
@@ -0,0 +1,51 @@
+help([[
+Load environment for running the GDAS application with Intel compilers and MPI.
+]])
+
+local pkgName    = myModuleName()
+local pkgVersion = myModuleVersion()
+local pkgNameVer = myModuleFullName()
+
+prepend_path("MODULEPATH", "/apps/dev/lmodules/core")
+
+load("PrgEnv-intel/8.2.0")
```

Comment:
@CoryMartin-NOAA 
We should open an issue to ensure we follow up on using `build.ver` and `run.ver` to describe the versions of the modules used in GDASapp.

Reply:
Sure. IIRC those aren't used on WCOSS, right? just RDHPCS?

Reply:
They are.  `load_fv3gfs_modules.sh` sources `run.ver`
We currently have `load_ufs_modules.sh` and `load_ufsda_modules.sh` that don't to allow for flexibility, but they could source the respective `ufs.run.ver` and `ufsda.run.ver`, this would make it easier to converge, rather than have 3 `load_xyz_modules.sh`

Reply:
We should be able to kill the DA module script when Orion goes to Rocky9, as then we will get rid of Conda.